### ISSUE TITLE
Stack offset of objects doesn't flip with Mirror mod in osu!standard

### ISSUE THREAD

**Issue by kionell** [ppy/osu#18511]
### Type

Game behaviour

### Bug description

As i said in the title, all stacked objects doesn't change their stack offset direction. This problem can result in a slight difference in difficulty and performance due to incorrect spacing between objects.

### Screenshots or videos

https://user-images.githubusercontent.com/43014304/171214817-a4369089-02ec-498e-b2cf-d9800cef0b94.mp4

https://user-images.githubusercontent.com/43014304/171214874-d3a69e2c-78c6-4fc8-8993-1d29ac1baa2d.mp4

![image](https://user-images.githubusercontent.com/43014304/171216765-80cb15eb-1f33-493c-8bf8-67824a885a6b.png)

![image](https://user-images.githubusercontent.com/43014304/171216898-96980310-20f4-4ba2-bba3-ef6a43a9df69.png)

### Version

2022.528.0

### Logs

-

**Comment by bdach** [comment]
I'm not even really sure that changing the offset is actually expected here? As in, if the mirror mod should do this, then so should hard rock, should it not? But hard rock doesn't seem to affect stack offset on stable at all.

**Comment by kionell** [comment]
Yeah, HR doesn't flip stacked objects either. The only problem in this is that it doesn't make any sense, since flipped beatmaps shouldn't differ in difficulty, should they?
